### ISSUE TITLE
Use Book Image for MediaMetadata if Available

### ISSUE THREAD

**Issue by OrangeDrangon** [izderadicka/audioserve-web#14]
`player.svelte` should use the current book image for `MediaMetadata` if it is available. It should fall back to the favicon if there is no book image. While not essential I think this will add some nice polish to the now playing notification.

**Comment by izderadicka** [comment]
Agreed,  the MediaSession API usage still needs bit more work - It does not work reliably on Chrome - I even created a bug for chromium  https://bugs.chromium.org/p/chromium/issues/detail?id=1337536.  On Android (at least on my current Samsung) it lacks  closer integration with native MediaSession,  so usability is problematic.  But definitely something we can try to improve in future.

Cover images for folders are little bit broader story - there is also some request for it in audioserve issues.  To implement it properly I think we will need to introduce new service for icons - so folder images are normalized, resized and cached. So full size images should not be loaded in cases where smaller icons are needed.

**Comment by OrangeDrangon** [comment]
They actually seem interested in fixing that big :). It is something I have noticed.

Dynamic image resizing by the server makes sense. 

Thanks as always for the awesome application.
